### PR TITLE
ci(jenkins): Customise login with an assistance message

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Enable a customized message on the Kibana login page.

## Why is it important?

Help to know the login details easily.

## Related issues

Caused by https://github.com/elastic/kibana/pull/51557
Closes https://github.com/elastic/apm-integration-testing/issues/677
Blocked by https://github.com/elastic/kibana/issues/53800

## How to test

```bash
python scripts/compose.py start 8.0.0   --build-parallel --no-apm-server-dashboards --no-apm-server-self-instrument --force-build
open http://localhost:5601
docker logs -f localtesting_8.0.0_kibana
```

## UI

image

### CODE REVIEW

File: scripts/modules/elastic_stack.py [hunk 1]
```diff
@@ -636,6 +636,8 @@ def __init__(self, **options):
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
         self.environment["ELASTICSEARCH_URL"] = ",".join(self.options.get(
             "kibana_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS])
+        if self.at_least_version("7.6"):
```
Comment (kuisathaverat) [inline scripts/modules/elastic_stack.py]: maybe also the link to the documentation for the rest of users https://github.com/elastic/apm-integration-testing#logging-in

**Comment by v1v** [comment]
jenkins run the tests please

File: scripts/modules/elastic_stack.py [hunk 2]
```diff
@@ -656,6 +656,11 @@ def __init__(self, **options):
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
         self.environment["ELASTICSEARCH_URL"] = ",".join(self.options.get(
             "kibana_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS])
+        if self.at_least_version("7.6") and options.get("xpack_secure"):
+            self.environment["XPACK_SECURITY_LOGINASSISTANCEMESSAGE"] = "Login\_details:\_`{}/{}`.\_" \
```
Comment (v1v) [inline scripts/modules/elastic_stack.py]: As I already described in the PR description, there is no other way to customize the login page using the env variables with docker-compose. That's the reason for using the `_` as a word separator.

Comment (cachedout) [reply scripts/modules/elastic_stack.py]: For people who come across this in the future, [this is the PR](https://github.com/docker/compose/pull/6923) that will eventually fix this in Docker Compose.

File: scripts/modules/elastic_stack.py [hunk 3]
```diff
@@ -656,6 +656,11 @@ def __init__(self, **options):
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
         self.environment["ELASTICSEARCH_URL"] = ",".join(self.options.get(
             "kibana_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS])
+        if self.at_least_version("7.6") and options.get("xpack_secure"):
+            self.environment["XPACK_SECURITY_LOGINASSISTANCEMESSAGE"] = "Login\\_details:\\_`{}/{}`.\\_" \
```
Comment (cachedout) [inline scripts/modules/elastic_stack.py]:
```suggestion
            self.environment["XPACK_SECURITY_LOGINASSISTANCEMESSAGE"] = "Login&#32;details:&#32;`{}/{}`.&#32;" \
```

File: scripts/modules/elastic_stack.py [hunk 4]
```diff
@@ -656,6 +656,11 @@ def __init__(self, **options):
...
+                "Further\\_details\\_[here]({}).".format(
```
Comment (cachedout) [inline scripts/modules/elastic_stack.py]:
```suggestion
                "Further&#32;details&#32;[here]({}).".format(
```

File: scripts/modules/elastic_stack.py (hunk 3 above)
Comment (v1v) [reply scripts/modules/elastic_stack.py]: Let me try this proposal

Comment (v1v) [reply scripts/modules/elastic_stack.py]: Changed -> https://github.com/elastic/apm-integration-testing/pull/702/commits/280e113546bc990d045a486c9ec0404437a34779

File: scripts/modules/elastic_stack.py (hunk 4 above)
Comment (v1v) [reply scripts/modules/elastic_stack.py]: Changed -> https://github.com/elastic/apm-integration-testing/pull/702/commits/280e113546bc990d045a486c9ec0404437a34779

**Pull request merged**